CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-# v 1.5.0 (?)
+# v 1.4.1 (?)
```

Comment:
You know master is expected to be a minor ahead of the released version? In other words after performing the patch release you'll have to minor bump this.